package.json

```diff
@@ -1,53 +1,35 @@
 {
   "name": "@percy-io/percy-webdriverio",
-  "version": "0.2.0",
+  "version": "1.0.0",
```

Comment:
@djones does this look like an ok major version bump? (Or should the version bump be done in a follow-up PR when the package gets released?)

Reply:
I'm ok with doing it in here.